src/env/env.h

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*   tosh - 21 Shell                                      ::::::::            */
+/*                                                      :+:    :+:            */
+/*   By: aholster <aholster@student.codam.nl>          +:+                    */
+/*       ivan-tey <ivan-tey@student.codam.nl>         +#+                     */
+/*       nloomans <nloomans@student.codam.nl>        +#+                      */
+/*                                                 #+#    #+#                 */
+/*   License: GPLv3                                ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef ENV_H
+# define ENV_H
+
+# include <libft.h>
+# include <stdlib.h>
+
+struct					s_env_list
+{
+	char				*key;
+	char				*value;
+	t_list_conn			conn;
+};
+
+typedef struct			s_env
+{
+	t_list_meta			meta;
+}						t_env;
+
+struct s_env_list		*unpack_env(t_list_conn *const node);
```

Comment:
There are not needed externally. Could you move them to a `private.h`?

A `typedef struct s_env t_env;` will be needed so that the outside would can still reference `t_env`.